batch_file_rename.sh

```diff
@@ -16,7 +16,7 @@
 
 funct_check_params()				# Start of the procedure
 {
-  if [ ${NARG} -lt 3 ]; then			# Check there are 3 arguments passed to the script, if not display the help 
+  if [ "${NARG}" -lt 3 ]; then			# Check there are 3 arguments passed to the script, if not display the help 

```

Comment:
This change shouldn't be needed, because the variable should be an integer. Rather than forcing this within the test, I would make the script more robust by first ensuring it's an integer and catching when it's not.